src/ontology/cl-edit.owl

```diff
@@ -27711,7 +27708,18 @@ AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023042 "L6 CT")
 AnnotationAssertion(oboInOwl:inSubset obo:CL_4023042 <http://purl.obolibrary.org/obo/cl#BDS_subset>)
 AnnotationAssertion(rdfs:label obo:CL_4023042 "L6 corticothalamic-projecting projecting glutamatergic cortical neuron")
 SubClassOf(obo:CL_4023042 obo:CL_4023013)
+SubClassOf(obo:CL_4023042 ObjectSomeValuesFrom(obo:RO_0013001 obo:UBERON_0010225))
 
+# Class: obo:CL_4023046 (corticothalamic VAL/VM Projecting glutamatergic neuron of the primary motor cortex (Mus musculus))
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:20556241") Annotation(oboInOwl:hasDbXref "PMID:9236245") obo:IAO_0000115 obo:CL_4023046 "A corticothalamic-projecting glutamatergic neuron that is located in L6 and lower L5 of the primary motor cortex, and is a thalamic projection pyramid with mostly untufted apical dendrites terminating in midcortical layers. CT VAL/VM cells have a near tonic firing pattern and are distinguished from L6 IT neurons by a lower inter-spike interval adaptation index.")
```

Comment:
I'm confused about "is a thalamic projection pyramid" in the second clause of the first sentence. Is there a way to assert that information without 'is a' to make it less ambiguous? We know that it is corticothalamic-projecting from the first clause, so is the intent to describe the morphology or the projection (or something else)?